lib/node_modules/@stdlib/stats/base/dmskrange/benchmark/benchmark.js

```diff
@@ -21,17 +21,37 @@
 // MODULES //
 
 var bench = require( '@stdlib/bench' );
-var randu = require( '@stdlib/random/base/randu' );
+var uniform = require( '@stdlib/random/array/uniform' );
+var bernoulli = require( '@stdlib/random/base/bernoulli' );
+var filledarrayBy = require( '@stdlib/array/filled-by' );
 var isnan = require( '@stdlib/math/base/assert/is-nan' );
 var pow = require( '@stdlib/math/base/special/pow' );
-var Float64Array = require( '@stdlib/array/float64' );
-var Uint8Array = require( '@stdlib/array/uint8' );
 var pkg = require( './../package.json' ).name;
 var dmskrange = require( './../lib/dmskrange.js' );
 
 
+// VARIABLES //
+
+var options = {
+	'dtype': 'float64'
+};
+
+
 // FUNCTIONS //
 
+/**
+* Returns either one or zero.
+*
+* @private
+* @returns {number} one or zero
+*/
+function rand() {
+	if ( bernoulli( 0.2 ) ) {
```

Comment:
```suggestion
	if ( bernoulli( 0.2 ) < 1 ) {
```

My  preference is to make the comparison explicit.

Reply:
But also, you don't need to do this. Just use `random/array/bernoulli` directly. A Bernoulli RV is either `0` or `1`, which is exactly what you want for the mask arrays.

Reply:
E.g., `bernoulli( len, 0.2, boptions )` where `boptions = { 'dtype': 'uint8' }`.

Reply:
ah well, there's something new to learn! pushing the required changes soon!